clients/web/src/views/widgets/HierarchyWidget.js

```diff
@@ -476,12 +476,16 @@ var HierarchyWidget = View.extend({
         this.$('.g-child-count-container').addClass('hide');
 
         var showCounts = _.bind(function () {
+            const folderCount = formatCount(this.parentModel.get('nFolders'));
             this.$('.g-child-count-container').removeClass('hide');
-            this.$('.g-subfolder-count').text(
-                formatCount(this.parentModel.get('nFolders')));
+            this.$('.g-subfolder-count').text(folderCount);
+            var tooltip = folderCount === 1 ? `${folderCount} total folder` : `${folderCount} total folders`;
```

Comment:
This would be cleaner as `const folderTooltip` (since we want to use `let` and `const`, not `var`).